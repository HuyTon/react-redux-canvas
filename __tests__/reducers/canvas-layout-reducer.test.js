// Reducer to be tested
import canvasLayoutReducer from '../../src/client/reducers/canvas-layout-reducer';
import {
  LOAD_CANVAS_LAYOUT,
} from '../../src/client/actions/action-types';

describe('LOAD_CANVAS_LAYOUT', () => {
  test('is correct', () => {
    const action = { type: LOAD_CANVAS_LAYOUT };

    expect(canvasLayoutReducer(undefined, action)).toMatchSnapshot();
  });
});