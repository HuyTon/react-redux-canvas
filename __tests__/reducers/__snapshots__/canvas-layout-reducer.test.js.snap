// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LOAD_CANVAS_LAYOUT is correct 1`] = `
Object {
  "title": undefined,
}
`;
